models/articles-model.js

```diff
@@ -70,7 +70,16 @@ function updateVote(newVote, article_id) {
         // }
         return result.rows[0]
     });
+}
 
+function removeCommentById(comment_id){
+    return db
+    .query(`DELETE FROM comments WHERE comment_id = $1 RETURNING *`, [comment_id])
+    .then(({rows}) => {
+        if(rows.length === 0) {
+            return Promise.reject({status: 404, msg: 'article does not exist'})
+        }
+    });
```

Comment:
Great! Nice use of the rows length to check if the delete has gone through successfully 👍 
One thing is though the error message should reflect that the comment id doesn't exist rather than the article